lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -80,6 +80,14 @@ class ReportUIFeatures {
   initFeatures(report) {
     this.json = report;
 
+    // TODO(i18n): Modifying Util singleton is not the best way to i18n this.
+    // Related TODO: https://github.com/GoogleChrome/lighthouse/pull/5713/files#r204948699
+    // Update util UIStrings, just like in report-renderer, we need to do it again b/c they have been reset.
+    const originalUIStrings = JSON.parse(JSON.stringify(Util.UIStrings));
```

Comment:
maybe we should encapsulate this as part of `updateAllUIStrings` like a companion function set or something?

Reply:
Like a `Util.cacheUIStrings()` then a `Util.resetUIStringsFromCache()` function tuple?

Reply:
yeah I was thinking something like that, just so that no one outside of `Util` has to muck with `UIStrings` property

Reply:
Done!